supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -218,6 +218,17 @@
 
 *See also*:
 
+[discrete]
+[[SmartNIC]]
```

Comment:
The anchor should be lowercase "smartnic" or "smart-nic". Can you also update the xrefs that use it when you change it?